curriculum/challenges/english/01-responsive-web-design/css-picasso-painting/part-020.md

```diff
@@ -0,0 +1,107 @@
+---
+id: 60b69a66b6ddb80858c5158a
+title: Part 20
+challengeType: 0
+dashedName: part-20
+---
+
+# --description--
+
+Now you have a large box. Give it a `border-top-color`, `border-right-color`, and `border-left-color` of `transparent`. Set the `border-bottom-color` to `GhostWhite`. This will make it look more like a hat.
```

Comment:
Does not look like a hat to me??
![image](https://user-images.githubusercontent.com/51722130/129191161-6e4dfc00-a03c-4e8b-bf51-b0c128f440b6.png)